pkg/controller/volume/persistentvolume/scheduler_binder.go

```diff
@@ -99,8 +98,9 @@ type SchedulerVolumeBinder interface {
 type volumeBinder struct {
 	ctrl *PersistentVolumeController
 
-	pvcCache PVCAssumeCache
-	pvCache  PVAssumeCache
+	nodeInformer coreinformers.NodeInformer
```

Comment:
Can you add a comment here that this nodeInformer cannot be used in FindPodVolumes?  Because the predicate is reused for cluster autoscaler, which passes in fake Node objects to the predicate.

Reply:
Added the comment in `FindPodVolumes` instead. Let me know if you disagree.